src/test/java/tutorial/TestTest.java

```diff
@@ -0,0 +1,7 @@
+package tutorial;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TestTest {
```

Comment:
this folder should not be here, you should make a folder in test with your use case name